src/assets/theme.css

```diff
@@ -1,30 +1,38 @@
-:root{
+:root {
     /* -------- Colors ----------*/
-    --clr-bg: #F5F5F5;
-    --clr-one: #F7C8E0; 
-    --clr-two: #DFFFD8;
-    --clr-three: #B4E4FF;
-    --clr-four: #95BDFF;
+    --clr-black: #000000;
+    --clr-white: #ffffff;
+    --clr-grey: #595959;

```

Comment:
@protibimbok `--clr-black`, `--clr-white`, `--clr-grey` etc. will be problematic if we decide to change the colors. Please update the color vars like this:

Text primary: Pure black (#000000)
Background primary: pure white (#FFFFFF)
Neutral: grey (#595959)
Accent color: #820097
Danger color: #F24E1E

Please let me know if you have any comments/suggestions about the naming.